app/js/actions/createLabOrder.js

```diff
@@ -0,0 +1,185 @@
+import 'babel-polyfill';
+import randomString, { generate } from 'randomstring';
+
+import loading from './loading';
+import axiosInstance from '../config';
+
+import {
+  CREATE_LAB_ORDER,
+  CREATE_LAB_ORDER_SUCCESS,
+  CREATE_LAB_ORDER_FAILURE,
+} from './actionTypes';
+
+const createLabOrderFailure = error => ({
+  type: CREATE_LAB_ORDER_FAILURE,
+  error,
+});
+
+const createLabOrderSuccess = data => ({
+  type: CREATE_LAB_ORDER_SUCCESS,
+  data,
+});
+
+const createLabOrder = labOrderData => async (dispatch) => {
+  try {
+    dispatch(loading(CREATE_LAB_ORDER, true));
+    const requestPayload = {
+      encounterProviders: [
+        {
+          encounterRole: '240b26f9-dd88-4172-823d-4a8bfeb7841f',
+          provider: 'f9badd80-ab76-11e2-9e96-0800200c9a66',
+        },
+      ],
+      encounterType: '7b68d557-85ef-4fc8-b767-4fa4f5eb5c23',
+      location: {
```

Comment:
This should just be a uuid that references a current location, and not create a whole new location.